SCMS.Services.Api/Brokers/Storages/IStorageBroker.StudentGuardian.cs

```diff
@@ -2,13 +2,16 @@
 // Copyright (c) Signature Chess Club & MumsWhoCode. All rights reserved.
 // -----------------------------------------------------------------------
 
+using System.Linq;
 using System.Threading.Tasks;
 using SCMS.Services.Api.Models.Foundations.StudentGuardians;
+using SCMS.Services.Api.Models.Foundations.Students;
 
 namespace SCMS.Services.Api.Brokers.Storages
 {
     public partial interface IStorageBroker
     {
         ValueTask<StudentGuardian> InsertStudentGuardianAsync(StudentGuardian studentGuardian);
+        public IQueryable<StudentGuardian> SelectAllStudentGuardians();
```

Comment:
All the methods in the interface are public by default. So we can skip it here. If you see it in other interfaces. please do a separate code rub too. Thanks.

Reply:
Please update the branch too.